pkg/cidata/cidata.TEMPLATE.d/boot/07-host-dns-setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -eux
+
+# Wait until iptables has been installed; 30-install-packages.sh will call this script again
+if command -v iptables >/dev/null 2>&1; then
+	if [ "${LIMA_CIDATA_UDP_DNS_LOCAL_PORT}" -ne 0 ]; then
```

Comment:
`[ -n ${...} ] && [ ${...} -ne 0 ]` might be more robust

Reply:
I'll change it, but the variable is set from an `int` in the template, so can't really be empty.